ai.js

```diff
@@ -1,67 +1,65 @@
 const axios = require('axios');
 
 async function callOpenApi(prompt, model, token) {
-    let returnStr = "";
-
-    await axios.post('https://api.openai.com/v1/completions',
-    {
-        "model": model,
-        "prompt": prompt,
-        "max_tokens": 1024
-    }, 
-    {
-        headers: {
-            Authorization: "Bearer " + token
-        },
-        timeout: 10000
-    }).then(res => {
-        returnStr = res.data.choices[0].text.trim().substring(0, 2000);
+    try {
```

Comment:
Why do you suggest using a standard `try-catch` block instead of the built-in `await-then-catch` structure that the Axios library provides?

Reply:
`.then` and `.catch` chaining are one interface for using promises, but you end up with lots of callbacks. Using a more standard `try`/`catch` block, your code is more linear and easier to follow, and you have fewer braces/`.then` chains to deal with when you start to need more promises or more operations.

Example
```js
const result = await fetch();
const json = await result.json();
// do something with json; another request can just follow linearly
```
versus
```js
fetch().then((result) => {
    return result.json();
}).then((json) => {
    // do something with json, but only in this nested context; heaven forbid you make another request and need more nested .thens
});
```